api/accesslog.proto

```diff
@@ -1,171 +0,0 @@
-syntax = "proto3";
```

Comment:
Even though it's not used now, can we keep this file? I think we will implement this at Lyft. If you want to have a comment somewhere that it's not used, that's fine, but I think it's useful to keep it around as we are thinking about implementations. (Perhaps you could just use it internally)

Reply:
That's fine with me.  Should we add the custom RPC configuration back as well with a comment?

Reply:
That sounds fine to me.  @htuch?

Reply:
I would probably just leave the file in the repo but change nothing about your change. That would be my vote.

Reply:
Yes, just leave the file then, we don't need the proto RPC config, just this if it will be more than just Google making use of it.